book/src/configuration/servers/README.md

```diff
@@ -220,6 +220,14 @@ Example. `["/msg NickServ IDENTIFY foo bar"]`.
 - **type**: array of string
 - **values**: array of any strings
 - **default**: not set
+  
+## `who_poll_enabled`
+
+Whether or not to WHO polling is enabled.
```

Comment:
I know it's a bit redundant, but users may tunnel vision on the configuration option they're looking for, so it might be good to tack on the "for servers without away-notify" here as well.

Reply:
Correct but I think that is an edge case I'm willing to accept.